### PR TITLE
refactor: Popconfirm cssinjs

### PR DESCRIPTION
**Pull request by zombieJ**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [x] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [x] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |     Refactor Popconfirm with cssinjs.      |
| 🇨🇳 Chinese |    重构 Popconfirm 为 cssinjs。     |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

File: components/_util/theme/interface.ts [hunk 1]
```diff
@@ -1,30 +1,31 @@
 import type * as React from 'react';
 import type { ComponentToken as AnchorComponentToken } from '../../anchor/style';
+import type { ComponentToken as BackTopComponentToken } from '../../back-top/style';
```
Comment (zombieJ) [inline components/_util/theme/interface.ts:3]: Prettier 现在还会自动 reorder，越来越像 eslint 了

Comment (MadCcc) [reply components/_util/theme/interface.ts:3]: 高级，排序确实像是 prettier 该做的

File: components/tooltip/style/index.tsx [hunk 2]
```diff
@@ -283,32 +288,41 @@ const genTooltipStyle: GenerateStyle<TooltipToken, CSSObject> = token => {
...
+    const tooltipArrowWidth = 8 * Math.sqrt(2); // FIXME: hardcode in v4
+    const tooltipShadowColor = new TinyColor('#000').setAlpha(0.07).toRgbString(); // FIXME: hardcode in v4
+    const tooltipBg = new TinyColor('#000').setAlpha(0.75).toRgbString(); // FIXME: hardcode in v4
+    const tooltipArrowRotateWidth =
+      Math.sqrt(tooltipArrowWidth * tooltipArrowWidth * 2) + tooltipArrowShadowWidth * 2; // FIXME: hardcode in v4
+
+    const TooltipToken = mergeToken<TooltipToken>(token, {
+      // default variables
+      tooltipMaxWidth: 250, // FIXME: hardcode in v4
+      tooltipColor: '#fff', // FIXME: hardcode in v4
```
Comment (MadCcc) [inline components/tooltip/style/index.tsx:309]: colorBgComponent?

File: components/popover/style/index.tsx [hunk 3]
```diff
@@ -109,8 +108,8 @@ const genBaseStyle: GenerateStyle<PopoverToken> = token => {
         color: popoverColor,
       },
 
-      // FIXME 没找到使用地方，先保留
-      '&-message': {
+      // Used for Popconfirm
```
[the remark this answers is not in the document]
Comment (zombieJ) [reply components/popover/style/index.tsx:111]: 是挺犹豫的，它的前缀还是 popover 如果挪到 popconfirm 就没前缀了

File: components/tooltip/style/index.tsx [hunk 4]
```diff
@@ -283,32 +288,41 @@ const genTooltipStyle: GenerateStyle<TooltipToken, CSSObject> = token => {
...
+export default (prefixCls: string, injectStyle: boolean): UseComponentStyleResult => {
+  const useOriginHook = genComponentStyleHook('Tooltip', token => {
+    // Popover use Tooltip as internal component. We do not need to handle this.
+    if (injectStyle === false) {
+      return [];
+    }
+
+    const { radiusBase, zIndexPopupBase } = token;
+    const tooltipArrowShadowWidth = 3; // FIXME: hardcode in v4
+    const tooltipArrowWidth = 8 * Math.sqrt(2); // FIXME: hardcode in v4
```
Comment (MadCcc) [inline components/tooltip/style/index.tsx:300]: token 里有个 arrowWidth 好像

File: components/popover/style/index.tsx (hunk 3 above)
[the remark this answers is not in the document]
Comment (MadCcc) [reply components/popover/style/index.tsx:111]: 先用 antCls 套下？

File: components/tooltip/style/index.tsx (hunk 2 above)
Comment (zombieJ) [reply components/tooltip/style/index.tsx:309]: 这只做 cssinjs，Tooltip 加了个跳过，Full Token 没有动。

File: components/tooltip/style/index.tsx (hunk 4 above)
Comment (zombieJ) [reply components/tooltip/style/index.tsx:300]: 见上，不用管 Tooltip 的 token

**Review by MadCcc — APPROVED** [review]

**Pull request merged**